### DISCUSSION TITLE
How can I get Galaxy Badge?

### DISCUSSION THREAD

**Discussion by fedirlut** [community/community#39816] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do I get the Galaxy Brain badge?
![173127571-f50a670c-5d34-44e2-9515-ce8b6fe25a69](https://user-images.githubusercontent.com/112725551/203494662-9e03abb6-c7f5-4f11-870f-0e0977e60b93.png)

**Comment by slava-shkarlet** [comment, marked as answer]
Hi @andrey-bogush ,

Below is a compiled knowledge of what I know, after hours of scouring through discussion forums, and trying to understand how the mechanics work (for some of the achievements at least).

YOLO - Merge a PR directly without review (can add a friend as a reviewer). Repo should be public.
Pull Shark - Merge at least 2 PRs into a public repo.
Galaxy Brain:
Answer at least 2 discussions.
Take note, you CANNOT mark your own answers to your own questions.
They have to be marked by another user. Discussions should be in a public repository.
Also if you mark answers to your own questions, there could be a bug whereby the achievement won't show despite having >=2 answers marked by other users.
Reference: https://github.com/orgs/community/discussions/18454
Also, please provide a couple of hours (or maybe up to a day) for your new achievements to show. If they haven't shown by then, perhaps you did not hit some "hidden" criteria. In most cases, developers seem to get their achievements instantly though.
Contact 'Github Support' if you face any issues.
The community can feel free to chime in, and provide their own observations on how the requirements of the new achievements, in case I may have missed some stuff out.

Thanks.

**Reply by fedirlut** [reply]
Thanks

**Comment by Veercodeprog** [comment]
to achieve the 'Galaxy Brain' achievement, take note of the following:

Answer at least 2 discussions.
Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
Discussions should be in a public repository.
There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Hazrat-Ali9** [comment]
How do I get a Galaxy Brain badge
Response
